typescript/modules/EDDAutocomplete.ts

```diff
@@ -90,7 +90,9 @@ export class BaseAuto {
     static initPreexisting(context?: Element | JQuery): void {
         $("input.autocomp", context).each((i, element) => {
             const visibleInput: JQuery = $(element);
-            const autocompleteType: string = $(element).attr("eddautocompletetype");
+            const autocompleteType: string = $(element).attr(
```

Comment:
I'm sort-of reverting this, so the full line is `const autocompleteType: string = $(element).data("eddautocompletetype");`. Note that `attr` is now [`data`](https://api.jquery.com/data/#data-html5).